common/build/eslint-config-fluid/eslint7.js

```diff
@@ -170,36 +162,30 @@ module.exports = {
         "unicorn/filename-case": [
             "error",
             {
-                "cases": {
-                    "camelCase": true,
-                    "pascalCase": true
-                }
-            }
+                cases: {
+                    camelCase: true,
+                    pascalCase: true,
+                },
+            },
         ],
         "unicorn/no-new-buffer": "error",
         "unicorn/no-unsafe-regex": "error",
 
         // eslint
         "arrow-body-style": "off",
-        "arrow-parens": [
-            "error",
-            "always"
-        ],
-        "camelcase": "off", // Superseded by @typescript-eslint/camelcase
+        "arrow-parens": ["error", "always"],
+        camelcase: "off", // Superseded by @typescript-eslint/camelcase
```

Comment:
This is something that bugs me... I would prefer the keys always be quoted, since that's consistent across all possible keys. Can prettier be configured to do that?

Reply:
I can check.

Reply:
They do indeed. https://prettier.io/docs/en/options.html#quote-props
I'll go ahead an update all the configs we have so far to ensure consistency.

Reply:
Yay!

Reply:
`consistent` seems great!

Reply:
Updated